Source/WarpX.H

```diff
@@ -436,6 +451,25 @@ public:
         const std::string& name,
         std::optional<const amrex::Real> initial_value);
 
+    /**
+     * \brief
+     * Allocate the MultiFab so that is like the specified MultiFab (same ba and dm)
+     * and optionally initialize it. This also adds the MultiFab
+     * to the map of MultiFabs (used to ease the access to MultiFabs from the Python
+     * interface
+     *
+     * \param mf[out] The MultiFab unique pointer to be allocated
+     * \param mf_model[in] The MultiFab to model
+     * \param name[in] The name of the MultiFab to use in the map
+     * \param initial_value[in] The optional initial value
+     */
+    static void AllocInitMultiFabFromModel (
```

Comment:
Would it be worth to introduce this in a separate PR?
(In order to make the current PR more focused.)

Reply:
This routine is only used in the implicit solver so would be an unused routine in a separate PR.

This routine may be useful elsewhere if other MultiFabs are created in a similar way, but I didn't look through the source for any such places.